examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +26,19 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
-
-    return true;
+	
+	int result;
+	
+	result = system(cmd);
+	
+	//false if error
+	if(result == -1)
```

Comment:
@ishassharmaa what would happen if the `cmd` returned an error code other than `-1`?